hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -39,7 +39,7 @@
 from hypothesis.internal.conjecture.shrinking.learned_dfas import SHRINKING_DFAS
 
 if False:
-    from typing import Dict  # noqa
+    pass
```

Comment:
This breaks our static type checking. And why would we want a `if False: pass` clause in our code..?

Reply:
Turns out that we actually `from typing import Dict` in the import section above, so deleting this would make sense - and I'll do that later - but leaving `pass` is pretty weird.